### DISCUSSION TITLE
How can I set font-size/weight in q-table label name?

### DISCUSSION THREAD

**Discussion by rodolfoserralha** [quasarframework/quasar#13503] [General - Components / Directives / etc] [answered]
Hello guys, i can't edit font-size/weight in label, when i do that taking the styles key, it seens to change all the column data font but not the label title.

Here is my table code and column data example

```
        <q-table
          title="Turmas Divergentes"
          :rows="rows"
          :columns="columns"
          row-key="name"
          rows-per-page-label="Turmas por página"
        />
```

```
  {
    name: "idTurma",
    label: "Turma Id",
    field: "idTurma",
    align: "left",
    sortable: true,
  },
```

![Captura de tela 2022-05-23 163308](https://user-images.githubusercontent.com/87541163/169892927-7772f4c5-80ec-45d5-a887-458674dffbfe.png)

**Comment by rodolfoserralha** [comment]
And also how can I set the header title in the middle? I tried margin and etc and nothing worked. Thanks!

**Comment by suleimanas** [comment, marked as answer]
headerStyle and headerClasses for th style and classes
or you can use header slot to customize the header row

**Reply by rodolfoserralha** [reply]
Thanks bro. It worked! Do you know how can I set the title header in the middle? Nothing worked.

**Reply by suleimanas** [reply]
Glad to help, the title is just a div outside the table.
try: 
`<q-table :title-class="text-center">...</q-table>`

or use slots
```
<q-table>
  <template v-slot:top="props">
        <div class="... text-center">Title</div>
  </template>
</q-table>
```
